### PR TITLE
SAR-6007 Add H1 within fieldset legend option to all radio helpers

### PR DESCRIPTION
**Pull request by EviNikolaidou**
## Checklist

* [x] I've included appropriate tests with any code I've added
* [x] I've executed the acceptance test pack locally to ensure there are no regressions
* [x] I've added my code using logical, atomic commits, squashing as appropriate - including the Jira issue number in the commit message
* [ ] I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/views/pages/completionCapacity.scala.html [hunk 1]
```diff
@@ -23,38 +23,36 @@
...
-                    "agent"     -> messages("pages.completionCapacity.agent"),
-                    "other"     -> messages("pages.completionCapacity.other")),
-                Seq("other" -> otherHidden),
-                '_idHidden -> "other",
-                '_classHidden -> "panel panel-indent",
-                '_labelClass -> "block-label radio-label",
-                '_legend -> Messages("pages.completionCapacity.description"),
-                '_legendClass -> "visually-hidden"
-            )
+        @inputRadioGroupHidden(
```
Comment (r-melvin) [inline app/views/pages/completionCapacity.scala.html:43]: Should this be the other helper? It'd make it clear that nothing should be hidden here

File: app/views/pages/employmentDetails/paysPension.scala.html [hunk 2]
```diff
@@ -23,27 +23,28 @@
 
 @main_template(title = title(Messages("pages.paysPension.title"), pensionsForm)) {
     @payeErrorSummary(
-        Messages("app.common.errorSummaryLabel"), pensionsForm, dataJourney=Some("CompanyPension")
+        Messages("app.common.errorSummaryLabel"), pensionsForm, dataJourney = Some("CompanyPension")
     )
 
-    <h1 class="form-title heading-large" id="pageHeading">@Messages("pages.paysPension.description")</h1>
+@*    <h1 class="form-title heading-large" id="pageHeading">@Messages("pages.paysPension.description")</h1>*@
```
Comment (r-melvin) [inline app/views/pages/employmentDetails/paysPension.scala.html:29]: This is commented out not deleted

File: app/views/helpers/templates/formHiddenYesNoRadio.scala.html [hunk 3]
```diff
@@ -23,6 +24,7 @@
             Seq(
             "true"->Messages("app.common.yes"),
             "false"->Messages("app.common.no")),
+            isHeading = true,
```
Comment (r-melvin) [inline app/views/helpers/templates/formHiddenYesNoRadio.scala.html:27]: this is always set to true and doesn't make use of the isHeading parameter passed in to the helper

File: app/views/pages/companyDetails/tradingName.scala.html [hunk 4]
```diff
@@ -39,31 +39,32 @@
...
 
-    <br>
-    <div class="inline form-group">
+        <br>
+        <div class="inline form-group">
         @formHiddenYesNoRadio(
             tradingNameForm,
             "differentName",
             Messages("pages.tradingName.description"),
+            true,
```
Comment (r-melvin) [inline app/views/pages/companyDetails/tradingName.scala.html:60]: You should always provide the name the boolean for context as well as the value. From reading this file only it's not possible to tell what the boolean is for

File: app/views/pages/employmentDetails/paidEmployees.scala.html [hunk 5]
```diff
@@ -23,36 +23,40 @@
...
-                paidEmployeesForm,
-                "alreadyPaying",
-                Messages("pages.paidEmployees.description"),
-                hiddenYesNoContent
-            )
+        @formHiddenYesNoRadio(
+            paidEmployeesForm,
+            "alreadyPaying",
+            Messages("pages.paidEmployees.description"),
+            true,
```
Comment (r-melvin) [inline app/views/pages/employmentDetails/paidEmployees.scala.html:56]: see above

File: app/views/pages/employmentDetails/willBePaying.scala.html [hunk 6]
```diff
@@ -56,28 +58,30 @@ <h1 class="form-title heading-large" id="pageHeading">@Messages("pages.willBePay
...
-                    '_labelClass -> "block-label",
-                    '_groupClass -> "inline"
-                )
-            }
+        @if(EmployingStaffForm.isRequiredBeforeNewTaxYear(now)) {
+            @formHiddenYesNoRadio(
+                willBePayingForm,
+                "willBePaying",
+                Messages("pages.willBePaying.description"),
+                true,
```
Comment (r-melvin) [inline app/views/pages/employmentDetails/willBePaying.scala.html:66]: as above

**Review by HarrNM — APPROVED** [review]

**Pull request merged**